strings/french_replacement.json

```
@@ -2,40 +2,44 @@
 
     "french": {
 		"shi": "shee",
-		"thi": "thee",
-		" yes ": " oui ",
-		"no": "non",
-       		 "I'm ": "j' ", 
-   	     	" am ": " suis ", 
-		" a ": " un ",
+		"thi": "zee",
+		"them ": "zem ",
+		"that ": "zat ",
+		"will": "weell",
+		"through": "srough",
+		"throw": "srow",
+		"though": "zough",
+		"thought": "sought",
```

```suggestion
		"thought": "zhoght",
```
Gonna be hard to distinguish "thought" and "sought" (as in "sought after") unless we apply accent magic